cpu/stm32f1/lpm_arch.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2013 INRIA
- * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
The lpm was only code stubs whit `/* TODO */`, I don't think it makes sense to use them since the file had no real content. I used instead the copyright by @DipSwitch, who is the actual author of the file ([stm32f2](https://github.com/RIOT-OS/RIOT/commits/master/cpu/stm32f2/lpm_arch.c))